python/cudf/cudf/io/dlpack.py

```diff
@@ -32,31 +31,19 @@ def from_dlpack(pycapsule_obj):
     or 2D.
     """
     try:
-        res, valids = cpp_dlpack.from_dlpack(pycapsule_obj)
+        res = libdlpack.from_dlpack(pycapsule_obj)
     except GDFError as err:
         if str(err) == "b'GDF_DATASET_EMPTY'":
             raise ValueError(
                 "Cannot create a cuDF Object from a DLPack tensor of 0 size"
             )
         else:
             raise err
```

Comment:
You can remove this entire error check block as the new libcudf APIs will throw if there's an exception.